docs/.vuepress/config.js

```diff
@@ -5,13 +5,36 @@ const sourceCodeLink = require('./containers/sourceCodeLink');
 const nginxRedirectsPlugin = require('./plugins/generate-nginx-redirects');
 const assetsVersioningPlugin = require('./plugins/assets-versioning');
 const extendPageDataPlugin = require('./plugins/extend-page-data');
-const { getBuildDocsVersion, getLatestVersion } = require('./helpers');
+const { getEnvDocsVersion, getEnvDocsFramework, TMP_DIR_FOR_WATCH, createSymlinks,
+  isEnvDev, getDocsFrameworkedVersions, getLatestVersion, FRAMEWORK_SUFFIX } = require('./helpers');
```

Comment:
This would probably be more readable if each imported prop would be in its own line?